packages/client/src/views/RecordAudit/ActionDetailsModal.tsx

```diff
@@ -452,6 +452,27 @@ const ActionDetailsModalListTable = ({
       width: 100
     }
   ]
+
+  const selectedCertificateTemplate = [
+    {
+      key: 'certTemplate',
+      label: intl.formatMessage(
+        certificateMessages.selectedCertificateTemplateLabel
+      ),
+      width: 200
+    }
+  ]
+
+  const certificateTemplateMessageDescriptor =
+    offlineData.templates?.certificates?.find(
```

Comment:
These types of `?` you can get rid of if you can trust certificates is always an array

Reply:
ok

Reply:
In this case, offlineData itself is a partial type, hence this optional chaininig here.

Reply:
That's true. You can still get rid of one of the optional fields so `.certificates?` to `.certificates` which itself is an improvement